Integrations/GitHub/GitHub.py

```diff
@@ -116,9 +107,9 @@ def get_issue_context(issue):
 
 
 def List_all_issues(show_all):
-    suffix = '/repos/{}/{}/issues'.format(OWNER, REPO)
+    suffix = SUFFIX['list']
     if show_all == 'false':
-        res = http_request('GET', suffix, None, None)
+        res = http_request('GET', suffix)
     else:
         res = http_request('GET', suffix, None, {'state': 'all'})  # include close issues
```

Comment:
no need to send the json param if you are not using it, just send the state dict as `params={'state': 'all'}`